utils/nctl/sh/node/join.sh

```diff
@@ -28,7 +28,7 @@ BID_DELEGATION_RATE=${BID_DELEGATION_RATE:-2}
 # ----------------------------------------------------------------
 
 # Await genesis era to complete.
-if [ "$(get_chain_era)" -eq 0 ]; then
+if [ "$(get_chain_era)" -lt 1 ]; then
```

Comment:
Maybe we can use the common `do_await_genesis_era_to_complete` function here?

Reply:
Done in [944cf35](https://github.com/casper-network/casper-node/pull/3906/commits/944cf355fd199ad84783c566d047e7ba1272b3b9).